File: 2963/2963.c
#include <stdio.h>

#define MAXN 10000

int main() {
    int N;
    int votos[MAXN];

    // Lê o número de candidatos.
    // Justificativa: "um total de N candidatos se inscreveram"
    scanf("%d", &N);

    // Lê os votos recebidos por cada candidato, na ordem de inscrição.
    // Justificativa: "relatório com N inteiros... em ordem de inscrição"
    for (int i = 0; i < N; i++) {
        scanf("%d", &votos[i]);
    }

    // Encontra o maior número de votos entre todos os candidatos.
    // Justificativa: "o mais votado será o novo bobo da corte"
    int maxVotos = votos[0];
    for (int i = 1; i < N; i++) {
        if (votos[i] > maxVotos) {
            maxVotos = votos[i];
        }
    }

    // Verifica se Carlos (primeiro inscrito, índice 0) tem o maior número de votos.
    if (votos[0] != maxVotos) {
        // Caso não tenha, ele não é eleito.
        printf("N\n");
    } else {
        // Caso esteja empatado ou tenha mais votos que os demais, ele vence por prioridade na inscrição.
        // Justificativa: "Caso haja empate entre um ou mais candidatos, 
        // aquele que tiver feito a inscrição primeiro é eleito."
        printf("S\n");
    }

    return 0;
}

File: 2963/2963.py
# Lista para armazenar o número de votos de cada candidato,
# de acordo com a ordem de inscrição.
# Justificativa: "A urna eletrônica gera um relatório com N inteiros, 
# correspondentes ao número de votos de cada candidato, ordenados pela ordem de inscrição."
votos = []

for i in range(int(input())):
    votos.append(int(input()))

# Encontra o maior número de votos entre todos os candidatos.
# Justificativa: "o mais votado será o novo bobo da corte"
maxVotos = max(votos)

# Verifica se o jovem Carlos (que é o primeiro da lista, índice 0) 
# não está entre os que têm o maior número de votos.
# Se ele não tiver o maior número de votos, ele não foi eleito.
if votos[0] != maxVotos:
    print('N')

# Caso Carlos esteja empatado com o maior número de votos,
# ele é eleito por ter se inscrito primeiro.
# Justificativa: "Caso haja empate entre um ou mais candidatos, 
# aquele que tiver feito a inscrição primeiro é eleito."
else:
    print('S')
